shopify/wishlist-public-remix-ssr/.template/config.json

```diff
@@ -0,0 +1,75 @@
+{
+  "version": "v1",
+  "setup": "./setup.md",
+  "webflow": {
+    "title": "Product Wishlists",
+    "description": "A complete wishlist solution for Shopify. Allows customers to create and manage multiple wishlists, and integrates seamlessly into the storefront and customer accounts.",
+    "content": "webflow/content.md",
+    "mainImage": "webflow/wishlist.webp",
+    "features": {
+      "shopify": true,
+      "frontend": true,
+      "models": true,
+      "actions": true
+    }
+  },
+  "featuredModels": {
+    "wishlist": {
+      "description": "Stores a named wishlist and associates it with a specific customer and shop.",
+      "featuredFields": ["name", "customer", "shop"]
+    },
+    "wishlistItem": {
+      "description": "Acts as a join table, linking a specific product variant to a wishlist.",
+      "featuredFields": ["variant", "wishlist", "customer"]
+    },
+    "shopifyCustomer": {
+      "description": "Extends the standard shopifyCustomer model to include relationships to all wishlists and wishlist items created by that customer.",
+      "featuredFields": ["wishlists", "wishlistItems"]
+    }
+  },
+  "featuredActions": {
+    "wishlist": {
+      "create": {
+        "description": "Allows a storefront customer to create a new, named wishlist for their account.",
+        "tags": ["added"]
+      },
+      "delete": {
+        "description": "Allows a storefront customer to delete one of their existing wishlists.",
+        "tags": ["added"]
+      }
+    },
+    "wishlistItem": {
+      "create": {
+        "description": "Adds a specific product variant to one of the customer's wishlists.",
+        "tags": ["added"]
+      }
+    }
+  },
+  "featuredBackends": {
+    "api/routes/proxy/wishlist/item/POST-create.ts": {
+      "description": "A backend proxy route that allows the storefront theme extension to add an item to a wishlist without a full user session.",
+      "tags": ["added"]
+    },
+    "api/actions/sendInStockEmail.ts": {
+      "description": "A global action that can be triggered to send an email notification to customers when an item on their wishlist comes back in stock.",
+      "tags": ["added"]
+    }
+  },
+  "featuredAccessControls": {
+    "shopify-storefront-customers": {
+      "wishlist": {
+        "description": "Grants permissions to logged-in storefront customers to fully manage their own wishlists (read, create, update, delete)."
+      }
+    }
+  },
+  "featuredFrontends": {
+    "extensions/wishlists-storefront/blocks/addToWishlist.liquid": {
+      "description": "A Shopify theme extension block that adds an 'Add to Wishlist' button and functionality to product pages.",
+      "tags": ["added"]
+    },
+    "extensions/wishlists/src/App.tsx": {
+      "description": "The main React component for the customer account UI. It renders the interface where customers can view and manage all their wishlists and the items within them.",
+      "tags": ["added"]
+    }
+  }
```

Comment:
Let's also mention `web/routes/_app._index.tsx` because there's an AutoForm that updates the `defaultUpdateFrequency` field.